src/components/dataset/data/DatasetDataView.tsx

```diff
@@ -144,8 +142,7 @@ function DatasetDataView({
           `#standardSQL
           SELECT ${DbColumns.ROW_ID},
             ${selectedTable?.columns?.map((column) => column.name).join(', ')} ${fromClause}
-            ${orderProperty ? `ORDER BY ${orderProperty} ${orderDirection}` : ''}
-          LIMIT ${QUERY_LIMIT}`,
+            ${orderProperty ? `ORDER BY ${orderProperty} ${orderDirection}` : ''}`,
```

Comment:
Remove query limit altogether -- With the limit in place, we were not able to page past that row # in the table.